setup.py

```diff
@@ -1,55 +1,61 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""The setup script."""
+
 from __future__ import with_statement
 
 try:
     from setuptools import setup, find_packages
 except ImportError:
     from distutils.core import setup, find_packages
 
-
 import grafana_inviter
 
-with open("README.md", "r") as fh:
-    readme = fh.read()
+
+with open("README.md") as readme_file:
+    readme = readme_file.read()
+
+with open("CHANGELOG.md") as changelog_file:
+    changelog = changelog_file.read()
+
+requirements = ["python-ldap", "requests", "anyconfig", "jsonschema"]
```

Comment:
No more version locking?

Reply:
Version locking is done in the requirements_dev.txt

Reply:
Actually this approach is pretty bad as when we deploy the package the concrete dependency versions are gone.
Will need to change that

Reply:
Pinning versions now